sign/pending_requests_test.go

```diff
@@ -3,6 +3,7 @@ package sign
 import (
 	"context"
 	"errors"
+	"fmt"
```

Comment:
shouldn't we use `log` instead of `fmt`?

Reply:
I think that's a leftover, thanks!